spring-cloud-stream/src/test/java/org/springframework/cloud/stream/binding/ChannelBindingServiceTests.java

```diff
@@ -94,10 +110,51 @@ public void testExplicitGroup() throws Exception {
 		MessageChannel inputChannel = new DirectChannel();
 		Binding<MessageChannel> mockBinding = Binding.forConsumer("foo", "fooGroup", Mockito.mock(AbstractEndpoint.class),
 				inputChannel, null);
-		Mockito.when(binder.bindConsumer("foo", "fooGroup", inputChannel, new Properties()))
+		when(binder.bindConsumer("foo", "fooGroup", inputChannel, new Properties()))
 				.thenReturn(mockBinding);
 		Binding<MessageChannel> binding = service.bindConsumer(inputChannel, name);
-		Assert.assertThat(binding, sameInstance(mockBinding));
+		assertThat(binding, sameInstance(mockBinding));
+
+		final AtomicReference<MessageChannel> dynamic = new AtomicReference<>();
```

Comment:
I'd make the second part a separate test, just to avoid a possible confusion. Also because it reuses the mockBinding.